src/sage/rings/finite_rings/finite_field_constructor.py

```diff
@@ -207,15 +207,20 @@ class FiniteFieldFactory(UniqueFactory):
 
     INPUT:
 
-    - ``order`` -- a prime power
-
-    - ``name`` -- string, optional.  Note that there can be a
-      substantial speed penalty (in creating extension fields) when
-      omitting the variable name, since doing so triggers the
-      computation of pseudo-Conway polynomials in order to define a
-      coherent lattice of extensions of the prime field.  The speed
-      penalty grows with the size of extension degree and with
-      the number of factors of the extension degree.
+    - ``order`` -- a prime power `p^n`, or a pair `(p,n)`
+
+    - ``name`` -- string or integer, optional. For a string,
+      ``name`` is the name of the generator over the prime
+      field. For convenience, in ``name`` is an integer, then
```

Comment:
```suggestion
      field. For convenience, if ``name`` is an integer, then
```
I had a typo. Feel free to set to positive review after fixing this and there are no (relevant) CI failures.